swagger_server/controllers/operator_controller.py

```diff
@@ -1,15 +1,27 @@
+import flask
+
 from swagger_server import util
+from swagger_server.models.evaluation import Operator
+from swagger_server.util import ValidationError
 
 
-def add_operator():  # noqa: E501
+def add_operator(payload):  # noqa: E501
     """Create an operator
 
     Creates an operator # noqa: E501
 
 
     :rtype: None
     """
-    return 'do some magic!'
+    if not payload:
+        raise ValidationError(400, 'Couldn\'t parse JSON POST body.')
+    if not payload.get('id'):
```

Comment:
I think you don't need to check for id because that's an autoincrement but you could validate the other fields on the Operator model (should be very similar to the add_operand method)